apis/field_error.go

```diff
@@ -137,6 +139,7 @@ func (fe *FieldError) getNormalizedErrors() []FieldError {
 	for _, e := range fe.errors {
 		errors = append(errors, e.getNormalizedErrors()...)
 	}
+	sort.Slice(errors, func(i, j int) bool { return errors[i].Message < errors[j].Message })
```

Comment:
What about same message different fields?

Reply:
That will have to be something added to flatten. I have been thinking about that...